example/js/lib/api/FixedDecimalFormatterOptions.mjs

```diff
@@ -106,17 +100,16 @@ export class FixedDecimalFormatterOptions {
 
         return new FixedDecimalFormatterOptions(diplomatRuntime.exposeConstructor, structObj);
     }
-#defaultConstructor() {
+    #defaultConstructor() {
         const diplomatReceive = new diplomatRuntime.DiplomatReceiveBuf(wasm, 8, 4, false);
-        
+
+
         const result = wasm.icu4x_FixedDecimalFormatterOptions_default_mv1(diplomatReceive.buffer);
-    
-        try {
-            return FixedDecimalFormatterOptions._fromFFI(diplomatRuntime.internalConstructor, diplomatReceive.buffer);
+
+        try {        return FixedDecimalFormatterOptions._fromFFI(diplomatRuntime.internalConstructor, diplomatReceive.buffer);
```

Comment:
issue

Reply:
I made sure to page through the diff locally before pushing.

I'll take a look when I have time